packages/statechannels-contracts/package.json

```diff
@@ -5,13 +5,13 @@
   "dependencies": {
     "@graphprotocol/common-ts": "1.0.1",
     "@openzeppelin/contracts": "3.2.2-solc-0.7",
-    "@statechannels/client-api-schema": "0.7.0",
-    "@statechannels/wallet-core": "0.10.0",
+    "@statechannels/client-api-schema": "0.8.0",
+    "@statechannels/wallet-core": "0.11.0",
     "lodash": "4.17.20"
   },
   "devDependencies": {
-    "@statechannels/devtools": "0.5.2",
-    "@statechannels/nitro-protocol": "0.13.0",
+    "@statechannels/devtools": "0.5.3",
+    "@statechannels/nitro-protocol": "0.14.0",
```

Comment:
These statechannels dependencies are split across production and development dependencies. Does that make sense? It makes automating the upgrade process a bit tricker (for annoying reasons); so I would be happy if they could all be of one kind or the other.